gordo/machine/machine.py

```diff
@@ -75,6 +75,23 @@ def __init__(
     def from_config(  # type: ignore
         cls, config: Dict[str, Any], project_name: str, config_globals=None
     ):
+        """
+        Construct an instance from a block of YAML config file which represents
```

Comment:
not from a yaml really, but from a dict which can come from the yaml

Reply:
appended a statement at the end, that it is to loaded in as a dict. :+1: